src/firebolt/model/engine.py

```diff
@@ -354,7 +354,9 @@ class _EngineUpdateRequest(FireboltBaseModel):
         # Update engine settings
         engine_settings_params = {
             "engine_type": engine_type,
-            "auto_stop_delay_duration": f"{auto_stop * 60}s" if auto_stop else None,
+            "auto_stop_delay_duration": f"{auto_stop * 60}s"
+            if auto_stop is not None
```

Comment:
While we're at it should we exclude negative numbers as well?

Reply:
So far we don't do any input validation. It is just when auto_stop was set to zero, which is a valid value, the if statement didn't allow to propagate it.